### PR TITLE
Add `log_level` option

### PR DESCRIPTION
**Pull request by felipecrs**

### CODE REVIEW

File: asterisk/rootfs/usr/share/tempio/logger.conf.gtpl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+console => notice,warning,error
+
+{{ else if .log_level == "info" }}
+console => notice,warning,error,verbose
+
+{{ else if .log_level == "debug" }}
+console => notice,warning,error,verbose,debug
+
+{{ else if .log_level == "trace" }}
+console => notice,warning,error,verbose,dtmf,fax
```
Comment (felipecrs) [inline asterisk/rootfs/usr/share/tempio/logger.conf.gtpl:23]:
```suggestion
console => notice,warning,error,verbose,debug,dtmf,fax
```

I think trace implies debug too.

**Comment by felipecrs** [comment]
@TECH7Fox awesome if you ask me. Should we merge?

**Comment by felipecrs** [comment]
I just found some issues in the template. May I fix them?

**Comment by TECH7Fox** [comment]
Ah I see, go ahead.

**Comment by felipecrs** [comment]
I think it's ready to go, directly to main branch. Let me just test it first. One second.

**Review by TECH7Fox — APPROVED** [review]

**Comment by felipecrs** [comment]
Seems to be working here. @pergolafabio if you have a chance someday, give it a try :).

**Comment by pergolafabio** [comment]
> Seems to be working here. @pergolafabio if you have a chance someday, give it a try :).

seems to be working, nice job!

**Pull request merged**